authorizer-app-backend/src/main/java/org/radarbase/authorizer/enhancer/ManagementPortalEnhancerFactory.kt

```diff
@@ -42,7 +42,8 @@ class ManagementPortalEnhancerFactory(
                     syncParticipantsIntervalMin = config.service.syncParticipantsIntervalMin,
                 ),
                 jwtResourceName = config.auth.jwtResourceName,
-                jwksUrls = config.auth.jwksUrls,
+                jwksUrls = config.auth.jwksUrls +
+                    config.auth.managementPortalUrl.trimEnd('/') + "/oauth/token_key",
```

Comment:
should we only add this when the jwksUrls are empty or null?

Reply:
Updated now 👍